app/assets/javascripts/components/question.coffee

```diff
@@ -18,11 +18,17 @@
     className
 
   render: ->
-    {text, answer_option_set} = @props.question
+    {text, intro, answer_option_set} = @props.question
     questionId = @props.question.id
 
     div
       className: @className()
+
```

Comment:
Ik vind het vrij bizar dat met deze newline coffeescript de p op de volgende regel nog meepakt als argument voor div. Het is dus niet alleen witruimtegevoelig maar juist indentatiegevoelig?